app/(main)/_components/_prizes/prizes-section.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+import Image from "next/image";
+
+import Prize from "@/app/(main)/_components/_prizes/prize";
+
+const PrizesSection = () => {
+  const [pantherHovered, setPantherHovered] = useState(false);
+  return (
+    <div
+      id="prizes"
+      className="relative flex flex-col w-full items-center justify-end gap-y-8 sm:gap-y-6 md:gap-y-4 pt-40 px-8 md:px-20 lg:px-32 overflow-x-clip"
```

Comment:
All sections have `py-40` added to the sections for the top and the bottom spacing. The prizes section lacks this, which causes it not to be uniform. We should add this back for an easier blend with the rest of the website.
```suggestion
      className="relative flex flex-col w-full items-center justify-end gap-y-8 sm:gap-y-6 md:gap-y-4 pt-40 px-8 md:px-20 lg:px-32 overflow-x-clip py-40"
```

Reply:
I agree! Let's do it